pkg/cloud/services/ec2/instances.go

```diff
@@ -1008,6 +1022,56 @@ func (s *Service) ModifyInstanceMetadataOptions(instanceID string, options *infr
 	return nil
 }
 
+// GetDHCPOptionSetDomainName returns the domain DNS name for the VPC from the DHCP Options.
+func (s *Service) GetDHCPOptionSetDomainName(ec2client ec2iface.EC2API, vpcID *string) *string {
+	fmt.Println("vpcID", vpcID)
+	log := s.scope.GetLogger()
+
+	if vpcID == nil {
+		log.Error(errors.New("vpcID is nil"), "vpcID is nil")
```

Comment:
```suggestion
		log.Info("vpcID is nil, skipping DHCP Option Set discovery")
```
I think an info level log makes more sense as we're letting the reconciler proceed, what do you think?

Reply:
+1